config/environments/development.rb

```diff
@@ -38,4 +38,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_host_name: "s3.eu-central-1.amazonaws.com",
+    s3_credentials: {
+      bucket: ENV['S3_BUCKET_NAME'],
+      access_key_id: ENV['AWS_ACCESS_KEY_ID'],
+      secret_access_key: ENV['AWS_SECRET_ACCESS_KEY']
+      },
+      url: ":s3_path_url",
+      path: "/:class/:attachment/:id_partition/:style/:filename"
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.